README.md

```diff
@@ -1,26 +1,39 @@
 # Grafana Status Panel
 
 This is a plugin meant to be used as a centralized view for the status of a server/cluster in a glance.
-It is very similar to the Single Stat panel, but it can hold multiple values and you can customize the thresholds for each of them separately.
+It is very similar to the Single Stat panel, but it can hold multiple values pulled from different data sources.
```

Comment:
It's not true that you can pull data from different data sources. You can make multiple queries to the same data source.